zerver/openapi/zulip.yaml

```diff
@@ -11374,14 +11425,24 @@ paths:
                         description: |
                           Present if `realm_linkifiers` is present in `fetch_event_types`.
 
-                          Array of objects where each object describes a single
+                          An ordered array of objects where each object describes a single
                           [linkifier](/help/add-a-custom-linkifier).
 
+                          The order of the array reflects the order that each
+                          linkifier is applied when linkifying messages and
+                          topics. By default, new linkifiers are ordered last.
+                          This order can be modified with [`PATCH
+                          /realm/linkifiers`](/api/reorder-linkifiers).
+
                           Clients will receive an empty array unless the event queue is
                           registered with the client capability `{"linkifier_url_template": true}`.
                           See [`client_capabilities`](/api/register-queue#parameter-client_capabilities)
                           parameter for how this can be specified.
 
+                          **Changes**: Changed in Zulip 8.0 (feature level 201).
+                          Linkifiers became consistently ordered with the order
+                          that they would be applied when rendering.
```

Comment:
Similarly as noted above, the tweaks to clarify that it's ordered are good (though I don't see a need to call out the order of new linkifiers here -- that's a product detail that's I think not relevant to correctly implementing the linkifier API). But we don't need to document a change here because as I understand it, there is no API client facing change to this parameter's behavior.

Reply:
Got it. Removed the changes section here. I kept the tweaks above just so it is consistent with the description elsewhere for linkifier arrays.